web/src/pages.rs

```diff
@@ -0,0 +1,30 @@
+use axum::response::{IntoResponse, Response};
+use db::users::User;
+
+use crate::templates::{Template, TemplatedPage};
+
+pub async fn home(t: Template, user: Option<User>) -> Response {
+    match user {
+        Some(user) => user_dashboard(t, user).await.into_response(),
+        None => landing(t).await.into_response(),
+    }
+}
+
+pub async fn landing(t: Template) -> TemplatedPage {
```

Comment:
I added this `t: Template` thing instead of calling the `template` method like we were doing when we paird!

Its _basically_ the same thing. We want to write the 'main' part of the page here, but want the extra stuff like the HTML header and script tags to be added for us!